### PR TITLE
search engine can be set to preference

### PR DESCRIPTION
**Pull request by PhantomLel**
For issue #2088

### CODE REVIEW

**Comment by PhantomLel** [comment]
why's that failing?

**Comment by Davidy22** [comment]
It should be possible to run actions on your own repo so that you don't need to wait for me to come round and hit the approve button on this for CI, I'll look back some time later when I remember to to see the result of this next run

**Comment by PhantomLel** [comment]
Ok thanks looks like I just had to run `black`

File: guake/data/prefs.glade [hunk 1]
```diff
@@ -135,7 +135,7 @@
                         <property name="can-focus">False</property>
                         <property name="halign">start</property>
                         <property name="margin-top">6</property>
-                        <property name="label" translatable="yes">&lt;span size="18000"&gt;&lt;b&gt;Guake properties&lt;/b&gt;&lt;/span&gt;</property>
```
Comment (Davidy22) [inline guake/data/prefs.glade:138]: Typo introduced?

File: guake/data/prefs.glade [hunk 2]
```diff
@@ -689,6 +698,49 @@
...
+                            <child>
+                              <object class="GtkBox">
+                                <property name="visible">True</property>
+                                <property name="can-focus">False</property>
+                                <property name="spacing">4</property>
+                                <child>
+                                  <object class="GtkLabel" id="lb_search_engine">
+                                    <property name="visible">True</property>
+                                    <property name="can-focus">False</property>
+                                    <property name="label" translatable="yes">Search Engine:                         </property>
```
Comment (Davidy22) [inline guake/data/prefs.glade:710]: Generally don't want to do spacing by just adding a bunch of spaces when doing UI work. I assume you did this to make a UI element line up somewhere, but if you want to do that you should use columns. The spaces are difficult for the translators to understand and match, just strip the spaces and if you want to do a re-layout that can be another pr

**Comment by Davidy22** [comment]
Added a release note file, the patch works, for the future it could do with a way for the user to add their own search engine but this'll do for now I think

**Review by Davidy22 — APPROVED** [review]
